ui/components-react/src/components-react/propertygrid/component/VirtualizedPropertyGrid.tsx

```diff
@@ -40,14 +40,21 @@ import { PropertyGridEventsRelatedPropsSupplier } from "./PropertyGridEventsRela
  * @public
  */
 export interface VirtualizedPropertyGridProps extends CommonPropertyGridProps {
+  /** Property grid view model that defines what needs to be rendered */
   model: IPropertyGridModel;
+  /** Handler of the events raised from the property grid */
   eventHandler: IPropertyGridEventHandler;
+  /** Property data provider used by the property grid */
   dataProvider: IPropertyDataProvider;
+  /** Properties for highlighting property data in the grid. */
   highlight?: HighlightingComponentProps & {
     filteredTypes?: FilteredType[];
```

Comment:
Since this is the only field remaining undocumented, can you add documentation to this last one ? (Same with the DataProvider below)
IE, will the `filteredTypes` be the only one highlighted or simply not highlighted.

Reply:
pushed the change - please have a look